### PR TITLE
Use FFMPEG.jl for video encoding

### PR DESCRIPTION
**Pull request by asinghvi17**
Use FFMPEG.jl for FFMPEG things.

### CODE REVIEW

**Comment by asinghvi17** [comment]
@ericphanson @daschw this should supercede #2095 - I've used FFMPEG.jl here and all the tests seem to be working, but there aren't any animation tests.  I'll add some.

File: test/runtests.jl [hunk 1]
```diff
@@ -93,3 +93,20 @@ end
         @test isequal(collect(zip(Plots.unzip(Point.(z))...)), z)
     end
 end
+
+@testset "Recording animations" begin
+    
+    for i in (gr, plotlyjs, pyplot, plotly)
```
Comment (daschw) [inline test/runtests.jl]: Plots does not test PyPlot and Plotly(JS) in the test suite.

**Comment by asinghvi17** [comment]
Cool, thanks for the info @daschw!  I put them there to make sure FFMPEG was working properly (which it isn't right now) but will change that shortly.  The problem is in FFMPEG, I'll quickly change that to another API and revert back.

**Comment by daschw** [comment]
Awesome!

**Comment by asinghvi17** [comment]
Perfect, it works now - I'll remove the check.

**Comment by ericphanson** [comment]
I think this closes https://github.com/JuliaPlots/Plots.jl/issues/2030

**Comment by asinghvi17** [comment]
Good catch, thanks!

**Comment by logankilpatrick** [comment]
Thanks for this @asinghvi17!

**Pull request merged**